pages/03.windows/02.installation/docs.en.md

```diff
@@ -58,6 +60,42 @@ If you need to uninstall or reinstall Adguard, do the following:
 
 ![](install_08_EN.png)
 
-In case regular uninstall doesn't work for any reason, you can try to use [Adguard Uninstall Tool](https://adguard.com/en/how-to-delete-adguard.html).
-
-To reinstall Adguard on your PC, launch the installation program again and repeat the steps in the Adguard Installation section.
\ No newline at end of file
+In case regular uninstall doesn't work for any reason, you can try to use Advanced method.
+
+To reinstall Adguard on your PC, launch the installation program again and repeat the steps in the Adguard Installation section.
+
+### Advanced
+First of&nbsp;all, you need to&nbsp;download the uninstaller tool created by&nbsp;our
+                        developers. You can do&nbsp;it&nbsp;by&nbsp;using
+                        <a target="_blank" href="https://cdn.adguard.com/public/Adguard/tools/Uninstall_Utility.zip">this
```

Comment:
Use markdown instead of the HTML for links and images:
https://guides.github.com/features/mastering-markdown/

```
Link: [link text](link href)
Image: ![image title](image source)
```